### PR TITLE
Improve patterns levels CY-4851

### PR DESCRIPTION
**Pull request by inesgomas**
I separated my changes in three commits:
- My suggestion to improve the level
  - I couldn't figure out if there was an automatic way to find out this group of rules / category they display [in their website](https://stylelint.io/user-guide/rules/list). Anyway, as a tool for css I left the codacy category as codestyle and created two severities: warn (https://stylelint.io/user-guide/rules/list/#possible-errors) & info (all the others)

- Doc generator result (Did I do this right?? 👀  A lot of files)
- Fixing tests

### CODE REVIEW

**Review by alerizzo — APPROVED** [review]
I'm approving the logic behind the new severities. No idea if the rest is ok, too many auto-generated changed files. (this also makes me wonder why and if it is really necessary to control autogenerated files here)

File: src/main/scala/codacy/stylelint/DocGenerator.scala [hunk 1]
```diff
@@ -48,14 +48,9 @@ object DocGenerator {
...
-    val enabled = DefaultPatterns.patterns.contains(patternName)
-    Pattern.Specification(
-      Pattern.Id(patternName),
-      Result.Level.Err,
-      Pattern.Category.CodeStyle,
-      None,
-      param,
-      enabled = enabled)
+    val enabled = CodacyValues.patternsEnabled.contains(patternName)
+    val level = if (CodacyValues.possibleErrorsPatterns.contains(patternName)) Result.Level.Warn else Result.Level.Info
```
Comment (machadoit) [inline src/main/scala/codacy/stylelint/DocGenerator.scala:52]: Even the `possibleErrorsPatterns`, we prefer to have them as Warning on our side? I am fine with it, just double checking :D

**Comment by machadoit** [comment]
Other than just double checking if we want them as Warning instead of errors, SGTM!

**Review by machadoit — APPROVED** [review]

File: src/main/scala/codacy/stylelint/DocGenerator.scala (hunk 1 above)
Comment (inesgomas) [reply src/main/scala/codacy/stylelint/DocGenerator.scala:52]: I just went by gut feel, while looking at the list of patterns they felt like they weren't Critical 😬 

@alerizzo Can you look into more detail in the list and share your opinion? https://stylelint.io/user-guide/rules/list/#possible-errors

**Pull request merged**